### PR TITLE
Added PHP Try Catch to Guide.

### PR DESCRIPTION
**Pull request by Gabehoverman**
- [X] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] None of my changes are plagiarized from another source without proper attribution.
- [X] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/english/php/try-catch/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+	//Code to handle the exception
+}
+
+
+```
+
+
+## Example
+
+Below is an example of how you could use a tru catch block in a real world situation:
```
Comment (co16353sidak) [inline guide/english/php/try-catch/index.md]: `tru` -> `try`

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
Just a small typographic error, otherwise good to merge.

@Gabehoverman Please make the required changes 

/cc: @RandellDawson

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**